plugin/web/extensions/sidebar/utils/file.ts

```diff
@@ -11,3 +11,8 @@ export const getNameFromPath = (path?: string) => {
 
   return path.split("/").slice(-1)[0];
 };
+
+export const joinNameWithExtension = (name?: string, extension?: string) => {
```

Comment:
Its up to you, but would createFilename, or buildFilename be a more appropriate name?

Reply:
Done!